docs/apache-airflow-providers-google/example-dags.rst

```diff
@@ -17,13 +17,13 @@
 
 Example DAGs
 ============
-
+docs/apache-airflow-providers-google/example-dags.rst
```

Comment:
Is this line meant to be there?

Reply:
it's possible that tests failed without it before or it was added automatically by pre-commit, 
but now tests passed without it